README.md

```diff
@@ -74,6 +79,13 @@ Simple test that checks whether a file or directory exists. This indicates it in
 ### get(path:  string):  Promise<string  |  Buffer>
 Gets a file as a string/Buffer.
 
+### activitiesGet(objectId: number | string, sort?: 'asc' | 'desc', limit?: number, sinceActivityId?: number):  Promise\<OcsActivity[]\>
+Returns all activities belonging to a file or folder.
+Use the `limit` argument to override the server-default.
+
+### usersGetUser(userId: string):  Promise\<OcsUser\>
```

Comment:
just `getUser` ?

Reply:
The other methods will be implemented at a later date.

Reply:
Then I'd recommend leaving them out for now. You'll cross that bridge when you get there

Reply:
Oh, I just realized that I misunderstood you. I apologize!

Reply:
Why name that `usersGetUser`?

Reply:
I misunderstood @kwisatz here.
My goal was to have all methods belong to a specific area to be grouped together.
I could make this `users: { get, list, setEnabled, etc }`. or something along those lines?

Reply:
Yeah, sorry, that's actually what I meant. My comment was about the naming, not the scope of functionality implemented.

Reply:
why not just getUsers though?